hwdb.d/60-evdev.hwdb

```diff
@@ -233,6 +233,13 @@ evdev:name:AlpsPS/2 ALPS DualPoint TouchPad:dmi:bvn*:bvr*:bd*:svnDellInc.:pnLati
  EVDEV_ABS_35=76:1815:22
  EVDEV_ABS_36=131:1330:30
 
+# Dell Latitude E5510
+evdev:name:AlpsPS/2 ALPS GlidePoint:dmi:*svnDellInc.:pnLatitudeE5510:*
+ EVDEV_ABS_00=73:1828:26
+ EVDEV_ABS_01=101:1319:27
+ EVDEV_ABS_35=73:1828:26
+ EVDEV_ABS_36=101:1319:27
```

Comment:
thanks, patch content looks good (see @bluca's comment though please) but the commit message needs fixing please. A simple `hwdb: add axis overrides for the Dell Latitude E5510 TouchPad` would be a good summary here.